cvat-ui/src/components/header/settings-modal/styles.scss

```diff
@@ -95,12 +96,20 @@
     }
 }
 
-.cvat-settings-page-back-button {
-    width: 100px;
-    margin-top: 15px;
+.cvat-player-settings-image-preview {
+    width: 100%;
+    max-height: 180px;
+    object-fit: cover;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5694342)